app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/integration/IntegrationOverviewHelper.java

```diff
@@ -45,7 +48,8 @@ private IntegrationOverviewHelper() {
 
     @SuppressWarnings("PMD.NPathComplexity")
     static IntegrationOverview toCurrentIntegrationOverview(final Integration integration,
-        final DataManager dataManager) {
+                                                            final DataManager dataManager,
+                                                            final OpenShiftConfigurationProperties openShiftProperties) {
```

Comment:
Passing the whole `OpenShiftConfigurationProperties` seems a bit much, perhaps we could refactor this class to be initialized with `OpenShiftConfigurationProperties` and make methods non-static.